ci/Jenkinsfile

```diff
@@ -38,26 +38,28 @@ pipeline{
         stage('setup TDX host'){
             steps{
                 node (tdxnode){
-                    script{
+                    script {
                         tdxnode = NODE_NAME
-                        workdir = pwd()
                         echo "tdxnode: $tdxnode, workspace: $workspace"
                         checkout scm
                         def distro = sh(script: ". /etc/os-release; echo \$NAME", returnStdout: true)
                         echo "system distro: ${distro}"
+                        dir ('ci'){
+                            workdir = pwd()
                         if (distro.contains('CentOS')){
-                            tdx_linux_stack_script = 'tdx_centos_linux_stack.sh'
-                        }
-                        sh "chmod +x $tdx_linux_stack_script"
-                        def statusCode = sh(script: "./$tdx_linux_stack_script --setuptdx", returnStatus:true)
-                        if (statusCode != 0 ){
-                           if(statusCode == 3){
-                                echo "Build machine will be restarted..."
-                                isRestartrequired = true
-                            }else {
-                                sh "exit 1"
+                                tdx_linux_stack_script = 'tdx_centos_linux_stack.sh'
+                            }
+                            sh "chmod +x $tdx_linux_stack_script"
+                            def statusCode = sh(script: "sudo ./$tdx_linux_stack_script --setuptdx", returnStatus:true)
+                            if (statusCode != 0 ){
+                               if(statusCode == 3){
+                                    echo "Build machine will be restarted..."
+                                    isRestartrequired = true
+                                }else {
+                                    sh "exit 1"
+                               }
                             }
-                       }
+                        }

```

Comment:
Fix the indentation. Should be on par with the above If statement.

Reply:
updated